src/components/collapsible/collapsible.jsx

```diff
@@ -76,7 +78,7 @@ class Collapsible extends PureComponent {
 
   renderContent() {
     return (
-      <div ref={ content => this.content = content }>
+      <div ref={ content => { this.content = content; } }>
```

Comment:
the reason for this is new linter settings, particularly [no-return-assign](http://eslint.org/docs/rules/no-return-assign).